requirements.txt

```
@@ -17,3 +17,4 @@ scikit_learn==1.7.0
 selenium==4.34.0
 tensorflow==2.18.0
 tf2onnx==1.16.1
+opencv-python==4.10.0.84
```

```suggestion
opencv-python==4.10.0.84
```
can we remove this? we have the dep already in the file? or should we keep it?